kafka-config.yml.sample

```diff
@@ -118,7 +118,7 @@ instances:
       # "consumer_group_1" we want to monitor "topic_1" with partitions 1, 2, and 3. We also want to monitor
       # "topic_2" within that same group, but want to monitor all partitions within that topic, so we'll give it a blank
       # set of partitions. Below is an example of the value for "consumer_group" field to achieve our desired configuration.
-      # {"consumer_group_1": {"topic_1": [1,2,3], "topic_2":[]}}
+      # {"consumer_group_1": '{"topic_1": [1,2,3], "topic_2":[]}}'
```

Comment:
@pe1227 

The single quote should enclose everything: topic group, topic name and partitions. Guess the correct syntax would be:
'{"consumer_group_1": {"topic_1": [1,2,3], "topic_2":[]}}'

The whole example is also a bit incoherent as it starts by saying `Say we wanted to monitor two consumer groups "consumer_group_1" and "consumer_group_2".` and the consumer_group_2 is then dropped from the example.
Also for the sake of consistency we should include the single quotes in the line `consumer_groups: <JSON Object whitelist of consumer groups to their topics and topics to their partitions, in which to collect consumer offsets for. Example form {"group_1":{"topic_1":[1,2]}}>`

In summary i suggest:

```
# Say we wanted to monitor two consumer groups "consumer_group_1" and "consumer_group_2".
# "consumer_group_1" we want to monitor "topic_1" with partitions 1, 2, and 3. We also want to monitor
# "topic_2" within that same group, but want to monitor all partitions within that topic, so we'll give it a blank
# set of partitions. Below is an example of the value for "consumer_group" field to achieve our desired configuration.
# '{"consumer_group_1": {"topic_1":[1,2,3], "topic_2":[]}, "consumer_group_2":{"topic_1":[]}}'
consumer_groups: <JSON Object whitelist of consumer groups to their topics and topics to their partitions, in which to collect consumer offsets for. Example form '{"group_1":{"topic_1":[1,2]}}'>
```